docs/operation/volume_management/volume_creation.rst

```diff
@@ -0,0 +1,95 @@
+Creating Volumes
+================
+
+Create **Volume** objects to use persistent storage in a MetalK8s cluster.
+
+Requirements
+------------
+
+:doc:`StorageClass Objects
+</operation/volume_management/storageclass_creation>` are registered in the
+cluster.
+
+Using the CLI
+-------------
+
+#. Create a **Volume** manifest.
+
+   You can define a new volume using the following template:
+
+   .. code-block:: yaml
+
+       apiVersion: storage.metalk8s.scality.com/v1alpha1
+       kind: Volume
+       metadata:
+         name: <volume_name>
+       spec:
+         nodeName: <node_name>
+         storageClassName: <storageclass_name>
+         rawBlockDevice:
+           devicePath: <device_path>
+
+   Set the following fields:
+
+   - **name**: the name of your volume, must be unique.
```

Comment:
That's more work than a comma can do. Without mucking up the capitalization for all these bullets, you best bet is: 
"the name of your volume (must be unique)"

No period here or in any subsequent bullet.